core/src/main/resources/META-INF/services/org.junit.jupiter.api.extension.Extension

```diff
@@ -0,0 +1,2 @@
+cz.xtf.core.context.TestCaseContextExtension
```

Comment:
So this is always registered?

According to https://github.com/xtf-cz/xtf/pull/490/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R164 it should be registered only when condition is success.

Reply:
As mentioned before automatic registration of extensions is not well possible with JUnit 5. `junit.jupiter.extensions.autodetection.enabled=true` must be set for `TestCaseContextExtension` to be automatically registered.

Reply:
Oh, sorry. I misread the README. Documentation is OK. Resolving.